back/engines/commercial/project_management/app/controllers/project_management/web_api/v1/moderators_controller.rb

```diff
@@ -0,0 +1,89 @@
+module ProjectManagement
```

Comment:
Essentially moved to the new engine. Don't know why that shows up as a new file. Sorry 😬